FV3/atmos_cubed_sphere/tools/fv_diagnostics.F90

```diff
@@ -1087,6 +1087,15 @@ subroutine fv_diag_init(Atm, axes, Time, npx, npy, npz, p_ref)
         Atm(n)%nudge_diag%nudge_t_dt(isc:iec,jsc:jec,npz) = 0.0
     endif
 
+    idiag%id_column_heating_nudge = register_diag_field('dynamics', &
+          'column_heating_nudge', axes(1:2), Time, &
+          'column integrated heating from nudging', &
+          'W/m**2', missing_value=missing_value)
```

Comment:
these are the expected units?

Reply:
Yeah, this term is multiplied by specific heat at constant pressure to get to units of W/m2, which is the conventional unit used to report column-integrated heating.